styles/twitch/catppuccin.user.css

```diff
@@ -247,14 +247,11 @@
       --color-text-button-disabled: @subtext0;
     }
 
-    &,
-    [class*="ScTokenOverrideCSSVars"] {
-      --color-text-link: @blue;
-      --color-text-link-active: @sky;
-      --color-text-link-focus: @sky;
-      --color-text-link-hover: @sky;
-      --color-text-link-visited: @lavender;
-    }
+    --color-text-link: @accent-color;
+    --color-text-link-active: @accent-color;
+    --color-text-link-focus: @accent-color;
+    --color-text-link-hover: @accent-color;
+    --color-text-link-visited: @accent-color;
```

Comment:
These need to be different, at least the first 3 - visited can be the same. You can use `darken(@accent-color, 5%)` for the first 3.